.github/workflows/common.yml

```diff
@@ -0,0 +1,35 @@
+name: common
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+env:
+  PACKAGEPATH: ${{ github.workspace }}/go/src/github.com/${{ github.repository }}
+  GOLANGCI_LINT_VERSION: v1.23.7
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x]
+        platform: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.PACKAGEPATH }}
+      - name: Install Go
+        uses: actions/setup-go@v2-beta
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Install golangci-lint
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ${{ github.workspace }}/bin ${{ env.GOLANGCI_LINT_VERSION }}
```

Comment:
what does `sh -s -- -b ${{ github.workspace }}/bin ${{ env.GOLANGCI_LINT_VERSION }}` do?

Reply:
`-b ${{ github.workspace }}/bin` is the directory where the binary is installed, and `${{ env.GOLANGCI_LINT_VERSION }}` is which version of the golangci-lint tool to install. I'll add a comment to make this clearer.

Reply:
sweet, thanks.